test/autoVerifyPsychologist.spec.ts

```diff
@@ -0,0 +1,31 @@
+import sinon from 'sinon';
+import verifyPsychologist from '../services/demarchesSimplifiees/verifyPsychologist';
+import autoVerifyPsychologists from '../services/demarchesSimplifiees/autoVerify';
+
+describe('autoVerifyPsychologist', () => {
+  let verifyPsychologistStub;
+
+  beforeEach(() => {
+    verifyPsychologistStub = sinon.stub(verifyPsychologist, 'verifyPsychologist');
+  });
+
+  afterEach((done) => {
+    verifyPsychologistStub.restore();
+    done();
+  });
+
+  // FIXME!
+  it.skip('Should call verifyPsychologist twice with correct args', async () => {
+    await autoVerifyPsychologists();
+
+    sinon.assert.callCount(verifyPsychologistStub, 2);
+
+    sinon.assert.calledWith(verifyPsychologistStub.getCall(0),
+      sinon.match(),
+      sinon.match());
+
+    sinon.assert.calledWith(verifyPsychologistStub.getCall(1),
+      sinon.match(),
+      sinon.match());
+  }).timeout(30000);
```

Comment:
The test was not set after refactoring of autoVerify. Do you want me to do it now OR will we fix it when adeli API will be back?

Reply:
we'll fix it later